### PR TITLE
fix(examples): fix broken examples due to upstream check in aws_kms

### PR DESCRIPTION
**Pull request by askulkarni2**
### What does this PR do?

Our examples broke due to [change](https://github.com/hashicorp/terraform-provider-aws/commit/360ace0e59081e57a5d07a102ba647b54bf3d2cf) in upstream provider.

`Error: "name" cannot begin with reserved AWS CMK prefix 'alias/aws'`

### More

- [ ] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/docs) for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: examples/analytics/spark-k8s-operator/k8s-schedular/pyspark-s3.yaml [hunk 1]
```diff
@@ -19,8 +19,8 @@ spec:
   mode: cluster
   image: "3bittechs/spark-py:3.0.1@sha256:dc1225d8429351426f05ba2dca85054368ec35f110601059c20d5587a029860f"
   imagePullPolicy: IfNotPresent
-  mainApplicationFile: s3a://aws001-preprod-dev-eks-emr-on-eks-logs/aws001-preprod-test-eks-emr-data-team-a/emr-data-team-a/taxidata/scripts/sample-spark-taxi-trip.py
-  arguments: ['s3a://aws001-preprod-dev-eks-emr-on-eks-logs/aws001-preprod-test-eks-emr-data-team-a/emr-data-team-a/taxidata/input/taxi-trip-data/', 's3a://aws001-preprod-dev-eks-emr-on-eks-logs/aws001-preprod-test-eks-emr-data-team-a/emr-data-team-a/taxidata/output/sparkk8soperator/taxi-trip-data/']
+  mainApplicationFile: s3a://apps001-preprod-dev-eks-emr-on-eks-logs/apps001-preprod-test-eks-emr-data-team-a/emr-data-team-a/taxidata/scripts/sample-spark-taxi-trip.py
```
Comment (Zvikan) [inline examples/analytics/spark-k8s-operator/k8s-schedular/pyspark-s3.yaml:22]: If this is from s3 bucket (seems like it), need to ensure that bucket name is corrected as-well.

**Review by Zvikan — APPROVED** [review]
LGTM, just small comment about the spark s3 bucket name

**Review by kcoleman731 — APPROVED** [review]
LGTM

**Comment by askulkarni2** [comment]
upstream provider is patched so we don't need to this workaround any more https://github.com/hashicorp/terraform-provider-aws/pull/24469

**Pull request closed without merging**